lib/iron_bank/local_records.rb

```diff
@@ -17,15 +17,15 @@ def self.directory
       IronBank.configuration.export_directory
     end
 
-    def self.export
+    def self.export(max_attempts = 180)
```

Comment:
maybe use named params instead so it's clear what the number would be when used by a client.
```rb
def self.export(max_attempts: 180)
  # ...
end
```
and usage would be one of:
```rb
IronBank::LocalRecords.export # defaults to 180
IronBank::LocalRecords.export(max_attempts: 260)
```
is `max_attempts` supposed to be time in seconds?